packages/contracts/src/addresses.ts

```diff
@@ -187,3 +191,146 @@ export const getCheckBalanceLowAddress = () => {
     solidityPackedKeccak256(['bytes'], [CheckBalanceLowArtifact.bytecode])
   )
 }
+
+/**
+ * Sorts an array of hex strings in ascending order and returns the sorted array. Does not mutate
+ * the original array.
+ *
+ * @param arr The array of hex strings to sort.
+ * @returns A new sorted array.
+ */
+export const sortHexStrings = (arr: Array<string>): Array<string> => {
+  // Create a copy of the array
+  const arrCopy = [...arr]
+
+  // Sort the copied array
+  return arrCopy.sort((a, b) => {
+    const aBigInt = BigInt(a)
+    const bBigInt = BigInt(b)
+    return aBigInt < bBigInt ? -1 : aBigInt > bBigInt ? 1 : 0
+  })
+}
+
+export const getGnosisSafeInitializerData = (
+  owners: Array<string>,
+  threshold: number
+): string => {
+  if (owners.length === 0) {
+    throw new Error(
+      "Sphinx: You must have at least one owner in your 'owners' array."
+    )
+  }
+  if (threshold === 0) {
+    throw new Error(
+      "Sphinx: You must set your 'threshold' to a value greater than 0."
+    )
+  }
```

Comment:
We don't rely on these checks on the website, so I haven't added tests for them. However, I don't see a strong reason to remove them.